assets/scss/main.scss

```diff
@@ -16,6 +16,7 @@
   --cds-button-primary-hover: #{carbon.$purple-80};
   --cds-button-primary: #{carbon.$purple-70};
   --cds-focus: #{carbon.$purple-70};
+  --cds-heading-01-font-weight: 500;
```

Comment:
This might have global style implications. Are we sure about it? Does it still adhere to our design guidelines?

Reply:
@eddybrando for this variable, we're trying to prevent the `bx-tab` label from truncating. The component is so restricted that we can't easily style it according to the team's needs.

The labels we've tried:
`Sampling a Bell Circuit` (too long)
`Sample a Bell Circuit` (too long)
`Sample a Bell State` (works, but only fits if we can adjust the font-weight)

https://user-images.githubusercontent.com/6276074/231824924-0586ef87-9343-488e-be41-3837c6fd5c85.mov


Any suggestions?

Reply:
Moved css variable in [this commit](https://github.com/Qiskit/qiskit.org/pull/3113/commits/f0f1ffd30b92da5afa00998a5ff17f41fea5e09b)